### DISCUSSION TITLE
tocic

### DISCUSSION THREAD

**Discussion by cschreib** [snitch-org/snitch#136] [On-boarding]
Onboarding checklist for @tocic:
 - [x] User satisfies the criteria for inclusion, as described in the [contribution documentation](https://github.com/snitch-org/snitch/blob/main/CONTRIBUTING.md), and has not been previously banned from this organization (you can search for their username in this discussion category, to see if they have been previously onboarded).
 - [x] User has been invited to the organization. To do so, head to the [People](https://github.com/orgs/snitch-org/people) tab of the organization, then select "Invite member". Type in their GitHub username, then select "Invite". In the next page, make sure that their role is selected as "Member", and that they are not yet invited into any team.
 - [x] User has accepted the invitation.
 - [x] User has responded to this discussion, confirming they have read the [code of conduct](https://github.com/snitch-org/snitch/blob/main/CODE_OF_CONDUCT.md) and the [contribution documentation](https://github.com/snitch-org/snitch/blob/main/CONTRIBUTING.md).
 - [x] User is added to the maintainer team.

**Comment by cschreib** [comment]
Hi @tocic! I'd like to invite you to become a maintainer, recognizing the interest and support you have provided to *snitch* since the library was created. I hope you don't mind serving as a bit of an experiment, since this is the first time this whole on-boarding process takes place. If you have any feedback on the process, please don't hesitate to share it here or in the [meta](https://github.com/orgs/snitch-org/discussions/categories/meta) channel.

**Comment by cschreib** [comment]
Thank you for accepting the invitation! If you could please have a read at the code of conduct and the contribution guidelines (see links above), in case you haven't already, and confirm here when you have done so. We can then finish the rest of the on-boarding process.

**Comment by tocic** [comment]
Yep, I've accepted the invitation, thanks for the trust! I also confirm that I've read the code of conduct and the contribution documentation.

**Reply by cschreib** [reply]
Thank you, and welcome to the team!

**Reply by tocic** [reply]
Hmm, looks like I still can't create branches:

```
Pushing to github.com:snitch-org/snitch.git
Enter passphrase for key '/home/tocic/.ssh/github_tocic': 
ERROR: Permission to snitch-org/snitch.git denied to tocic.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Reply by cschreib** [reply]
Ah yes, the "maintainer" team wasn't given write access; try again?

**Reply by tocic** [reply]
It works now!

**Reply by cschreib** [reply]
By the way, you may want to [make your membership public](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership). The default is for it to be private, and only you can change it. This is not mandatory, but it would probably help others gain confidence in joining if they see there are other members.

I will add this suggestion to the checklist.
